networks/rpc/http.go

```diff
@@ -232,6 +235,18 @@ func NewFastHTTPServer(cors []string, vhosts []string, timeouts HTTPTimeouts, sr
 	handler := newCorsHandler(srv, cors)
 	handler = newVHostHandler(vhosts, handler)
 
+	if nrAppName != "" && nrLicense != "" {
```

Comment:
Don't we need to notify user if only one variable is set?

Reply:
@ehnuje I changed the logic to print a critical log if only one variable is set. Thanks!